plutus-contract-certification/NOTICE

```diff
@@ -0,0 +1,14 @@
+Copyright 2019 Input Output (Hong Kong) Ltd.
```

Comment:
```suggestion
Copyright 2022 Input Output (Hong Kong) Ltd.
```
I guess there is a policy to add such files, if so, better use the actual year of creation...